### PR TITLE
(maint) manage hiera conf in lookup test

### PR DESCRIPTION
**Pull request by johnduarte**
This commit updates the v3_config_and_data test to ensure that it
manages the hiera.yaml config if it exists.

### CODE REVIEW

**Comment by joshcooper** [comment]
I think @Magisus is looking at this too

**Comment by Magisus** [comment]
Josh told me the same issue also exists in v4_hieradata_with_v5_configs.rb.

**Comment by Magisus** [comment]
👍

**Comment by Magisus** [comment]
Wait, this needs to target stable, I think.

File: acceptance/tests/lookup/v3_config_and_data.rb [hunk 1]
```diff
@@ -12,9 +12,13 @@
     confdir = master.puppet('master')['confdir']
     codedir = master.puppet('master')['codedir']
 
+    step "backup global hiera.yaml" do
+      on(master, "cp #{confdir}/hiera.yaml /tmp/C99629-hiera.yaml.bak", :acceptable_exit_codes => [0,1])
```
Comment (joshcooper) [inline acceptance/tests/lookup/v3_config_and_data.rb]: Could you change the backup destination to use `host.tmpfile` so we don't have to assume `/tmp`?

**Comment by johnduarte** [comment]
With the above changes, the lookup tests run successfully on Ubuntu 14.04
```
be beaker -h log/latest/hosts_preserved.yml -t tests/lookup

...
              - Test Case Summary for suite 'tests' -
       Total Suite Time: 149.29 seconds
      Average Test Time: 24.88 seconds
              Attempted: 6
                 Passed: 6
                 Failed: 0
                Errored: 0
                Skipped: 0
                Pending: 0
                  Total: 6

```

**Pull request merged**